app/views/static_pages/contact.html.erb

```diff
@@ -0,0 +1,6 @@
+<% provide(:title, "Contact") %>
+
+<h1>Contact</h1>
+<p>
+Contact the Ruby on Rails Tutorial about the sample app at the
+<a href="https://www.railstutorial.org/contact">contact page</a>. </p>
```

Comment:
Please insert a new line here.  Please do this for all files marked with the 🛑 symbol in GitHub